ui/raidboss/data/03-hw/raid/a8n.js

```diff
@@ -169,10 +184,19 @@
       response: Responses.tankBuster(),
     },
     {
-      id: 'A8N Long Needle',
+      id: 'A8N Long Needle Party',
       netRegex: NetRegexes.headMarker({ id: '003E' }),
+      condition: () => data.me !== data.bruteTank,
       response: Responses.stackOn(),
     },
+    {
+      id: 'A8N Long Needle Active Tank',
```

Comment:
Hmm, not every long needle comes with a flarethrower.  Also in thinking about this, I guess saying `tank cleave` here conflicts with the flarethrower timeline trigger too?  Do you have any thoughts about how to reconcile all of that?

Reply:
I don't think Flarethrower is a conflict, really. Looking at this more closely, we could probably just check whether we're on an even or odd count for Long Needle? Something like

```
{
  condition: function(data) {
    data.needleCount = data.needlecount || 0;
    return needleCount % 2 === 1 && [OTHER EXISTING CONDITIONS];
  }
 //do stuff
  run: function(data) {
    data.needlecount += 1;
  }
}
```
And then for the party Long Needle, we can check whether the user is a tank, and if they are, then reverse the condition from the tank Long Needle trigger. It's not trivial, but it's completely doable.

Reply:
Is there a possibility for a phase skip here that would skip one such that a counter wouldn't work? Do we need something like a flarethrower timeline trigger that sets a `flarethrowerSoon = true` and then maybe say "Don't Stack" if that's true? Sorry to make this more complicated.

Reply:
Well, I can't say for certain that there isn't, but all of my logs seem to indicate that the initial block from Brute Justice appearing until the intermission is 100% fixed. I've never seen an early push into the intermission, and indeed it's highly possible to completely skip it in party finder.

That aside, maybe a better way to handle it would be to enable/disable based on usage of Super Jump and J-Kick? The only dangerous Long Needle happens after J-Kick and before Super Jump, OR at the very start of the rotation. So we can start with the Long Needle "stay out" active, and then once we see Super Jump, disable it until J-Kick?

Reply:
That seems like a reasonable approach if you leave some comments about what's going on.  Thanks!

Reply:
Alright, thanks for the feedback on this. I think this should better handle those possibilities.